### PR TITLE
Set 'displaylogo' to false by default in the plotConfig setting.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #1600.

### CODE REVIEW

File: frontend/taipy-gui/src/components/Taipy/Chart.tsx [hunk 1]
```diff
@@ -506,18 +506,22 @@ const Chart = (props: ChartProp) => {
...
             try {
-                plconf = JSON.parse(props.plotConfig);
+                plconf = {...JSON.parse(props.plotConfig), removeLogo};
             } catch (e) {
                 console.info(`Error while parsing Chart.plot_config\n${(e as Error).message || e}`);
             }
             if (typeof plconf !== "object" || plconf === null || Array.isArray(plconf)) {
                 console.info("Error Chart.plot_config is not a dictionary");
-                plconf = {};
+                plconf = removeLogo;
```
Comment (FredLL-Avaiga) [inline frontend/taipy-gui/src/components/Taipy/Chart.tsx]: Why not add it once line 525 ?

File: frontend/taipy-gui/src/components/Taipy/Chart.tsx [hunk 2]
```diff
@@ -506,18 +506,22 @@ const Chart = (props: ChartProp) => {
     }, [props.figure, selected, data, config, dataKey]);
 
     const plotConfig = useMemo(() => {
+        const removeLogo = {displaylogo: false};
         let plconf: Partial<Config> = {};
         if (props.plotConfig) {
             try {
-                plconf = JSON.parse(props.plotConfig);
+                plconf = {...JSON.parse(props.plotConfig), removeLogo};
```
Comment (FredLL-Avaiga) [inline frontend/taipy-gui/src/components/Taipy/Chart.tsx]: This wouldn't work
You need to expand the object

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

**Comment by FabienLelaquais** [comment]
@FredLL-Avaiga 
But my implementation breaks a valid test. So...

File: frontend/taipy-gui/src/components/Taipy/Chart.tsx [hunk 3]
```diff
@@ -518,6 +518,10 @@ const Chart = (props: ChartProp) => {
                 plconf = {};
             }
         }
+        else {
+            plconf = {};
+        }
+        plconf.displaylogo = plconf.displaylogo || false;
```
Comment (FredLL-Avaiga) [inline frontend/taipy-gui/src/components/Taipy/Chart.tsx]: `plconf.displaylogo = !!plconf.displaylogo` has the advantage of forcing the value to a boolean
and `else` should be on the same line as the `if` }

**Review by FredLL-Avaiga — DISMISSED** [review]

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**